src/mesh/mesh_refinement.cpp

```diff
@@ -41,6 +41,7 @@ namespace parthenon {
 
 // TODO(Yurlunger) Identify an idiomatic way of applying reflective
 // boundary conditions in the boundary condition refactor code
+/*
```

Comment:
We should figure this out and either take this out or enable it if necessary.

Reply:
I vote for taking it out for now. Whether or not it's necessary requires understanding the restart issue. It does not appear necessary at runtime. Which implies to me that it probably doesn't belong here.

Reply:
Removed. Tests in our downstream code appear to show the downstream restart problem is no longer present.